install/ci-vm/ci-linux/startup-script.sh

```diff
@@ -9,24 +9,28 @@ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BF
 echo "deb https://download.mono-project.com/repo/ubuntu stable-focal main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list
 sudo apt update
 apt install mono-complete -y
+apt-get install -y libtesseract-dev
 
 mkdir repository
 cd repository
 
 # Use gcsfuse and import required files
-mkdir temp TestFiles TestResults
+mkdir temp TestFiles TestResults vm_data reports
 gcsfuse --only-dir=TestData/ci-linux/ spdev temp
 cp temp/* ./
 umount temp
 rmdir temp
 
 vm_name=$(curl http://metadata.google.internal/computeMetadata/v1/instance/hostname -H "Metadata-Flavor: Google")
 vm_name=(${vm_name//./ })
-mkdir vm_data reports
 
-sudo gcsfuse --implicit-dirs --only-dir vm_data/${vm_name} spdev vm_data
-sudo gcsfuse --implicit-dirs --only-dir TestFiles spdev TestFiles
-sudo gcsfuse --implicit-dirs --only-dir TestResults spdev TestResults
+echo "spdev   /repository/vm_data         gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=vm_data/${vm_name}  0 0" | sudo tee -a /etc/fstab
+echo "spdev   /repository/TestFiles     gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=TestFiles  0 0" | sudo tee -a /etc/fstab
+echo "spdev   /repository/TestResults     gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=TestResults  0 0" | sudo tee -a /etc/fstab
```

Comment:
This is for the VM test instances, not the sample-platform deployment, right? I just want to be double sure.

Reply:
Yes it is for VM instances.